src/main/scala/bitlap/sbt/analyzer/action/SbtDependencyAnalyzerGoToAction.scala

```diff
@@ -57,16 +57,9 @@ final class SbtDependencyAnalyzerGoToAction extends DependencyAnalyzerGoToAction
     val declared = DependencyUtils.getDeclaredDependency(module)
     declared
       .find(dc =>
-        if (
-          DependencyScopeEnum.values.exists(d =>
-            d.toString
-              .toLowerCase() == dc.getCoordinates.getVersion.toLowerCase || Constants.PROTOBUF == dc.getCoordinates.getVersion.toLowerCase
-          )
-        ) {
-          dc.getCoordinates.getArtifactId == coordinates.getArtifactId && dc.getCoordinates.getGroupId == coordinates.getGroupId
-        } else {
-          dc.getCoordinates.equals(coordinates)
-        }
+        // hard code, see SbtDependencyUtils#getLibraryDependenciesOrPlacesFromPsi
+        (dc.getCoordinates.getArtifactId == coordinates.getArtifactId || dc.getCoordinates.getVersion == coordinates.getArtifactId) &&
```

Comment:
Putting the artifact in the version position while traversing the expr is just a temporary decision